### ISSUE TITLE
Update .htaccess to Apache 2.4 syntax

### ISSUE THREAD

**Issue by gonatan** [CONTENIDO/CONTENIDO#96]
Some settings in .htaccess files have to be updated to be compatible w/ Apache 2.4.

E.g.: in htdocs/data/.htaccess
```
# Prevent access to this folder
Order Deny,Allow
Deny from all
```
vs.
```
# Prevent access to this folder
Require all denied
```

See https://httpd.apache.org/docs/2.4/upgrading.html for details.

It would be great if the syntax for both versions could be given. This can be achieved as follows (see https://serverfault.com/a/650183):

```
<IfModule mod_authz_core.c>
   # Apache 2.4
</IfModule>
 
<IfModule !mod_authz_core.c>
   # Apache 2.2
</IfModule>
```

The alternative below is more straightforward but requires the Apache module mod_version which would introduce an unwanted dependency for CONTENIDO:

```
<IfVersion >= 2.4>
   # Apache 2.4
</IfVersion>

<IfVersion < 2.4>
   # Apache 2.2
</IfVersion>
```

**Comment by tone2tone** [comment]
Using the "IfVersion" module functionality might indeed cause problems. I am not that experienced with htaccess stuff, BUT in some of my webhosting products, I fail to get the exact version of  Apache on php level if Apache config is set to ServerTokens "Prod" - httpd.conf denies to send out the exact version. Calls for `apache_get_version()` or `$_SERVER['SERVER_SOFTWARE']` just return "Apache", no further version information given. 
If this restricts htaccess to NOT get exact version info as well, relying on "IfVersion" (an extension) might be a bad idea, whilst "IfModule" is a core function.

**Comment by gonatan** [comment]
Right, the Apache server signature can be configured for hardening reasons using e.g.
* `ServerSignature On | Off`
* `ServerTokens Full | OS | Minimal | Minor | Major | Prod`

But these settings do not interfere with the `IfVersion` functionality AFAIK. Still the Apache module `mod_version` will become an additional and probably unwanted dependecy. So we should stick w/ the first solution even if does not read as good as the latter one.

BTW the PHP version signature can be configured using `expose_php Off` for the same purpose (see https://crycode.de/apache-serversignatur-anpassen)

**Comment by tone2tone** [comment]
@gonatan They DO interfere. As I say, server settings CAN prevent to give you the exact version number of Apache, and there seems no way around it. Therefore, "IfVersion" is basically out of the picture IMHO. If you find ANY approach to prove me wrong. There is no workaround to get an information that a server (for good reason) doesn't want to give away. We can't force users to reconfigure Apache Servers to our needs...
